backend/src/main/java/com/woowacourse/kkogkkog/presentation/CouponController.java

```diff
@@ -23,18 +18,6 @@ public CouponController(CouponService couponService) {
         this.couponService = couponService;
     }
 
-    @PostMapping
-    public ResponseEntity<Void> createCoupon(@RequestBody CouponCreateRequest couponCreateRequest) {
-        Long couponId = couponService.save(couponCreateRequest);
-        return ResponseEntity.created(URI.create("/api/coupons/" + couponId)).build();
-    }
-
-    @GetMapping
-    public ResponseEntity<CouponsResponse> showCoupons() {
-        CouponsResponse couponsResponse = couponService.findAll();
-        return ResponseEntity.ok(couponsResponse);
-    }
-
     @GetMapping("/{couponId}")
     public ResponseEntity<CouponResponse> showCoupon(@PathVariable Long couponId) {
```

Comment:
```suggestion
    public ResponseEntity<CouponResponse> show(@PathVariable Long couponId) {
```

기본적인 단일 조회는 `show`를 사용하기로 했던 것 같아요~! 한번 확인해주세요 🙏